source/manual/howto-setup-maintenance-mode.html.md

```diff
@@ -0,0 +1,78 @@
+---
+owner_slack: "#govuk-developers"
+title: Setup maintenance mode
+section: Frontend
+layout: manual_layout
+parent: "/manual.html"
+related_repos: [frontend, government-frontend]
+---
+
+## What is maintenance mode?
+
+Maintenance mode is when a public-facing application displays a user-friendly message to inform users about planned downtime such as a database upgrade instead of displaying unintended error pages.
+
+## Setup maintenance mode
+
+There are two main steps involved:
+
+1. Create a maintenance flag in the app views
+2. Enable/Disable the maintenance mode
+
+## Create a maintenance flag
+
+Note: This is a one-time setup that is done in the application
+
+1. Set up the initializer file to read the environment variables
+
+Rails.application.config.maintenance_mode = ENV["MAINTENANCE_MESSAGE"].present?
+Rails.application.config.maintenance_message = ENV["MAINTENANCE_MESSAGE"]
+
+Example PR [https://github.com/alphagov/frontend/pull/5072]
```

Comment:
To convert the text into a link we can update this to:
`[Example PR](https://github.com/alphagov/frontend/pull/5072)`